alf/examples/networks/impala_cnn_encoder.py

```diff
@@ -162,14 +163,28 @@ def create(input_tensor_spec,
             image in the mini batch.
         kernel_initializer: initializer for the Conv2D and FC layers in the
             network.
-    
+        apply_image_scale_transform: when set to ``True``, the input image will
+            first go through a scale transformation so that the pixels are
+            converted from 0 - 255 integer to 0.0 - 1.0 float.
+
     Returns:
-        
+
         A module representing the Impala CNN encoding network.
 
     """
     stacks = []
-    last_output_spec = input_tensor_spec
+    if apply_image_scale_transform:
```

Comment:
Perhaps derive apply_image_scale_transform from input_tensor_spec? (i.e. if spec.dtype is uint8

Reply:
Good idea. Let me update it.